WordPress/gutenberg
# Feat: Update textControl to searchControl in taxonomy search.

## What?
Fixes: #64593 

## Why?
The "Search categories" field in the Post sidebar uses a normal TextControl even though it's a search field.

## How?
This PR updates the TextControl component to `SearchControl` in sidebar taxonomy search.

## Testing Instructions
- Create any post.
- View the post settings in Sidebar and in categories it should be SearchControl now instead of normal TextControl.
